web/src/components/deployments-detail-page/log-viewer/index.tsx

```diff
@@ -96,6 +126,8 @@ export const LogViewer: FC = memo(function LogViewer() {
   const dispatch = useAppDispatch();
   const [handlePosY, setHandlePosY] = useState(maxHandlePosY - INITIAL_HEIGHT);
   const logViewHeight = maxHandlePosY - handlePosY;
+  const [skipTargetId, setSkipTargetId] = useState<string | null>(null);
```

Comment:
not sure why we need this variable, we can use `activeStage.Id` to get the current stage id directly.

Reply:
I'm sorry if I'm misunderstanding this.
I use these properties for also opening and closing a dialog but please give a advice if there are better implementations.

Reply:
That makes sense and is pretty good!
Let me apply it 🙌

Reply:
🙌